### PR TITLE
M-1 Create a project for Manager

### PR DESCRIPTION
**Pull request by Boryszs**

### CODE REVIEW

**Review by alek134 — APPROVED** [review]
lgtm

File: mvnw.cmd [hunk 1]
```diff
@@ -0,0 +1,188 @@
+@REM ----------------------------------------------------------------------------
```
Comment (mateuszjanczak) [inline mvnw.cmd:1]: Is this file needed?

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.7.0</version>
+		<relativePath/>
+	</parent>
+	<groupId>pl.move2.it</groupId>
+	<artifactId>manager</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<name>manager</name>
+	<description>We want to exchange knowledge and learn new things in our free time</description>
```
Comment (mateuszjanczak) [inline pom.xml:15]: That is our motto, but does it describe this application well?

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+			<artifactId>junit-jupiter-engine</artifactId>
+			<version>5.8.1</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.mockito</groupId>
+			<artifactId>mockito-core</artifactId>
+			<version>4.6.1</version>
+			<scope>test</scope>
+		</dependency>
```
Comment (mateuszjanczak) [inline pom.xml:39]: Which testing tool will we use? JUnit with Mockito or Groovy with Spock?

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-maven-plugin</artifactId>
+			<version>2.7.0</version>
+		</dependency>
+		<dependency>
+			<groupId>org.glassfish.jaxb</groupId>
+			<artifactId>jaxb-runtime</artifactId>
+		</dependency>
```
Comment (mateuszjanczak) [inline pom.xml:63]: What is the purpose of this package?

File: pom.xml [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+			<groupId>org.codehaus.groovy</groupId>
+			<artifactId>groovy-all</artifactId>
+			<version>2.4.7</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>javax.validation</groupId>
+			<artifactId>validation-api</artifactId>
+			<version>2.0.1.Final</version>
+		</dependency>
```
Comment (mateuszjanczak) [inline pom.xml:80]: What is the purpose of this package?

File: README.md [hunk 6]
```diff
@@ -1 +1,44 @@
...
+
+Come in ManagerApplication click run on left strap or use  
+Ctrl + Shift + F10
+
+
+
+### Command line run test:  
+`
+mvn spring-boot:run
+`
```
Comment (mateuszjanczak) [inline README.md:35]: Are you sure this command is running the tests?

File: README.md [hunk 7]
```diff
@@ -1 +1,44 @@
...
+Change name which one you prefer click ok
+
+You can run project click Run or shortcut Shift + F10
+
+### Way 2 
+
+Come in ManagerApplication click run on left strap or use  
+Ctrl + Shift + F10
+
+
```
Comment (mateuszjanczak) [inline README.md:30]: Remove unnecessary lines

File: README.md [hunk 8]
```diff
@@ -1 +1,44 @@
-# manager
\ No newline at end of file
+# manager
+
+### Technology:
```
Comment (mateuszjanczak) [inline README.md:3]: "Technologies"?

File: README.md [hunk 9]
```diff
@@ -1 +1,44 @@
...
+    -Java version 11
+    -Spring Boot
+    -Junit 5
+    -Spring Security 
+    -Eureka Server Netflix
```
Comment (mateuszjanczak) [inline README.md:8]: Is it still used?

File: README.md [hunk 10]
```diff
@@ -1 +1,44 @@
-# manager
\ No newline at end of file
+# manager
+
+### Technology:
+    -Java version 11
+    -Spring Boot
+    -Junit 5
+    -Spring Security
```
Comment (mateuszjanczak) [inline README.md:7]: This service will be available internally, which means that we do not need to secure it for now

File: pom.xml (hunk 5 above)
Comment (Boryszs) [reply pom.xml:80]: It allows you to validate incoming data

File: mvnw [hunk 11]
```diff
@@ -0,0 +1,316 @@
+#!/bin/sh
```
[the remark this answers is not in the document]
Comment (Boryszs) [reply mvnw:1]: this file is not needed

File: mvnw.cmd (hunk 1 above)
Comment (Boryszs) [reply mvnw.cmd:1]: this file is not needed

File: pom.xml [hunk 12]
```diff
@@ -0,0 +1,117 @@
...
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.7.0</version>
+		<relativePath/>
+	</parent>
+	<groupId>pl.move2.it</groupId>
```
[the remark this answers is not in the document]
Comment (Boryszs) [reply pom.xml:11]: I change to pl.move2.it probably it will probably be better

File: pom.xml (hunk 2 above)
Comment (Boryszs) [reply pom.xml:15]: It's motto for description the work of the organization

File: pom.xml (hunk 3 above)
Comment (Boryszs) [reply pom.xml:39]: we should choose Groovy with Spock

File: pom.xml (hunk 4 above)
Comment (Boryszs) [reply pom.xml:63]: I remove this dependency

File: pom.xml (hunk 5 above)
Comment (Boryszs) [reply pom.xml:80]: I remove this dependency it will not be of any use now

File: README.md (hunk 6 above)
Comment (Boryszs) [reply README.md:35]: Correct is the maven test, thanks for your vigilance.

File: README.md (hunk 8 above)
Comment (Boryszs) [reply README.md:3]: You're right

File: README.md (hunk 9 above)
Comment (Boryszs) [reply README.md:8]: No it's are wrong list

File: README.md (hunk 10 above)
Comment (Boryszs) [reply README.md:7]: Ok

File: pom.xml (hunk 12 above)
[the remark this answers is not in the document]
Comment (mateuszjanczak) [reply pom.xml]: So we leave it as it is? I don't see any change

File: pom.xml (hunk 3 above)
Comment (mateuszjanczak) [reply pom.xml]: Ok, so could you please create example test in Spock? Like 2 + 2 = 4?

Comment (Boryszs) [reply pom.xml]: Ok I will add

File: src/test/java/it/move2/manager/ManagerApplicationTests.java [hunk 13]
```diff
@@ -0,0 +1,13 @@
+package it.move2.manager;
```
Comment (mateuszjanczak) [inline src/test/java/it/move2/manager/ManagerApplicationTests.java:1]: This test is no longer necessary

File: pom.xml (hunk 3 above)
Comment (mateuszjanczak) [reply pom.xml]: I have run 'mvn test' and it didn't detect any tests

**Review by Corel-Cormen — APPROVED** [review]

**Review by PrzemyslawKuca — APPROVED** [review]

**Review by mateuszjanczak — APPROVED** [review]
Good job ;D
